### PR TITLE
Add a repeating type of `until`

### PR DESCRIPTION
**Pull request by kierenbeckett**
### What is the context of this PR?
Add a repeating type of `until`.  See also ONSdigital/go-launch-a-survey#106 and ONSdigital/eq-schema-validator#67.

### How to review 
Run `test_routing_repeat_until.json` , fill in some names and choose yes/no for the more people question at various stages.  See that you get asked about the sex of all the people's names you entered.  Note that the summary page only displays the first person in the repeating group, fixing this isn't within the scope of this PR.

### CODE REVIEW

File: scripts/test_schemas.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@ if [ "$1" == "--local" ] || [ "$2" == "--local" ]; then
 fi
 
 if [ "$run_docker" == true ]; then
-    docker pull onsdigital/eq-schema-validator
-    validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
+    docker pull onsdigital/eq-schema-validator:add_repeat_until
```
Comment (insacuri) [inline scripts/test_schemas.sh:9]: Note: remember to remove this before merging into master

**Review by insacuri — APPROVED** [review]
Looks reasonable.  Though I think 2 others should still look at this because there's probably some detail I'm not understanding with my newness

File: app/questionnaire/questionnaire_schema.py [hunk 2]
```diff
@@ -191,7 +191,7 @@ def answer_is_in_repeating_group(self, answer_id):
         group = self.get_group(block['parent_id'])
 
         repeat_rule = self.get_repeat_rule(group)
-        return repeat_rule and repeat_rule['type'] in ['answer_count', 'answer_count_minus_one', 'answer_value']
```
Comment (nealedj) [inline app/questionnaire/questionnaire_schema.py:194]: 👍

**Review by nealedj — APPROVED** [review]
Nicely done!

**Pull request merged**